assets/js/supplement_reminders/supplement_reminders_view.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from "react";
+import { AddSupplementReminderView } from "./add_supplement_reminder";
+import { SupplementReminderTable } from "./supplement_reminder_table";
+
+export class SupplementRemindersView extends Component {
```

Comment:
You might not need to do this then ... empty constructor. Probably try it stateless.